docs/manual/src/kotlin/lifetimes.md

```diff
@@ -0,0 +1,14 @@
+# Kotlin Lifetimes
+
+All interfaces exposed via Kotlin expose a public API for freeing
+the Kotlin wrapper object in lieu of reliable finalizers. This is done
+by making the "base class" for all such generated objects implement the
+`Disposable` and `AutoCloseable` interfaces.
+
+As such, these wrappers all implement a `close()` method, which must be
+explicitly called to ensure the associated Rust resources are reclaimed.
+
+The best way to arrange for this to be called at the right time is beyond
+the scope of this document; you should consult the official documentation for
+`AutoClosable`, but one common pattern is the Kotlin
+[use](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/use.html) function.
```

Comment:
I would probably recommend:

```suggestion
[`use` function](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/use.html).
```

just the small word `use` is a bad click target.